src/main/java/org/folio/entlinks/service/messaging/authority/InstanceAuthorityLinkUpdateService.java

```diff
@@ -89,7 +89,14 @@ private void processEventsByChangeType(List<AuthorityChangeHolder> changeHolders
 
   private void prepareAndSaveAuthorityDataStats(List<AuthorityChangeHolder> changeHolders) {
     var authorityDataStats = changeHolders.stream()
-      .map(AuthorityChangeHolder::toAuthorityDataStat)
+      .map(authorityChangeHolder -> {
+        var stat = authorityChangeHolder.toAuthorityDataStat();
+        InventoryEvent event = authorityChangeHolder.getEvent();
+        if (event.getNew() != null) {
+          stat.setStartedByUserId(event.getNew().getId());
```

Comment:
`id` in `new` is an id of the authority record. You have to use, `metadata.updatedByUserId` field of `new`